### PR TITLE
EASY-1922 respond with service is running when curling to the service baseurl

### PR DESCRIPTION
**Pull request by stephanHogenboom**
Fixes EASY-1922

#### When applied it will
* respond with "EASY Sword2 Service running.." when you curl to the service baseurl

### CODE REVIEW

File: src/main/scala/nl.knaw.dans.easy.sword2/servlets/EasySword2Servlet.scala [hunk 1]
```diff
@@ -19,12 +19,12 @@ import java.io._
...
 
-class HelloServlet extends HttpServlet {
+class EasySword2Servlet extends HttpServlet {
   @throws(classOf[ServletException])
   @throws(classOf[IOException])
-  override def doGet(request: HttpServletRequest, response: HttpServletResponse) {
+  override def doGet(request: HttpServletRequest, response: HttpServletResponse): Unit = {
     val out: PrintWriter = response.getWriter
-    out.println("hello, world")
+    out.println("EASY Sword2 Service running...")
```
Comment (rvanheest) [inline src/main/scala/nl.knaw.dans.easy.sword2/servlets/EasySword2Servlet.scala]: great change.
As an improvement, would it be possible to add the version number as well?
See also https://github.com/DANS-KNAW/easy-deposit-api/blob/f3467c347dd103b5fda4b5e35822bb09e795b360/src/main/scala/nl.knaw.dans.easy.deposit/servlets/EasyDepositApiServlet.scala#L28.

If you feel like it, add it to the other services as well 😜

Comment (stephanHogenboom) [reply src/main/scala/nl.knaw.dans.easy.sword2/servlets/EasySword2Servlet.scala]: Sure, i will look into it

File: src/main/scala/nl.knaw.dans.easy.sword2/ApplicationWiring.scala [hunk 2]
```diff
@@ -77,12 +77,14 @@ class ApplicationWiring(configuration: Configuration) extends DebugEnhancedLoggi
       .toMap
 
     SampleTestDataEnabled(sampleDir, sampleRates)
-  } else SampleTestDataDisabled
+  }
+                       else SampleTestDataDisabled
```
Comment (stephanHogenboom) [inline src/main/scala/nl.knaw.dans.easy.sword2/ApplicationWiring.scala:81]: strange auto format

File: src/main/scala/nl.knaw.dans.easy.sword2/servlets/EasySword2Servlet.scala (hunk 1 above)
Comment (stephanHogenboom) [reply src/main/scala/nl.knaw.dans.easy.sword2/servlets/EasySword2Servlet.scala]: Created an issue and corresponding pull requests

**Review by rvanheest — APPROVED** [review]

**Pull request merged**